install_config/router/f5_router.adoc

```diff
@@ -237,7 +237,7 @@ $ oadm router --external-host-partition-path=/OpenShift/zone1 ...
 This section reviews how to set up F5 native integration with {product-title}.
 The concepts of F5 appliance and {product-title} connection and data flow of F5
 native integration are discussed in the
-xref:../../architecture/additional_concepts/f5_big_ip.adoc#architecture-f5-native-integration[F5
+xref:../../architecture/topics/f5_big_ip.adoc#architecture-f5-native-integration[F5
```

Comment:
@gaurav-nelson - this link won't work. You probably don't need to change it here as a redirect for this will take care of it. 

https://docs.openshift.com/container-platform/3.6/architecture/additional_concepts/f5_big_ip.html
>>
https://docs.openshift.com/container-platform/3.6/architecture/networking/network_plugins.html#architecture-additional-concepts-f5-big-ip
(although not directly to that anchor as I am pretty sure you can't forward to or from anchors, just links).

But it highlights how we link to modular topic areas going forward. Let's discuss in the next meeting.

@openshift/team-documentation